### PR TITLE
chore: upgrade pf for Integration Detail Activity components

### PR DESCRIPTION
**Pull request by kahboom**
relates to [ENTESB-12896](https://issues.redhat.com/browse/ENTESB-12896)

- drop `patternfly-react` for `@patternfly/react-core` and `@patternfly/react-table` in Integration Detail Activity components
- update rendering integration detail activity `steps` in `syndesis` module to accommodate changes in `ui` package
- remove unused component `IntegrationDetailActivityItemStep`
- fix affected stories, minor styling fix

Screenshot 2020-03-06 15 43 15

Screenshot 2020-03-06 15 43 27

### CODE REVIEW

**Comment by gashcrumb** [comment]
Looking good!  Would you mind maybe seeing if you can improve the table layout a bit using some [width modifiers](https://www.patternfly.org/v4/documentation/react/components/table#with-width-modifiers) to pad out the step, time and status columns?

File: app/ui-react/packages/ui/src/Integration/Activity/IntegrationDetailActivityItem.tsx [hunk 1]
```diff
@@ -1,152 +1,177 @@
...
+
+    return newRows.reduce((a: any, b: any) => a.concat(b), []);
+  };
+
+  const columns = [
+    i18nHeaderStep,
+    i18nHeaderTime,
+    i18nHeaderDuration,
+    i18nHeaderStatus,
+    i18nHeaderOutput
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Integration/Activity/IntegrationDetailActivityItem.tsx]: I think this is where you'd add those modifiers to increase the width of a few of those columns, i.e.

`const columns = [ { title: i18nHeaderStep, transform: cellWidth(12) }, etc etc ];`

Comment (kahboom) [reply app/ui-react/packages/ui/src/Integration/Activity/IntegrationDetailActivityItem.tsx]: btw I remember now that I tried this last week and it seems to have no effect on the cell width, not sure why. Would you be able to test this locally? I've tried variations of the following:

```
const columns = [
    { title: i18nHeaderStep, transform: cellWidth(10) },
    { title: i18nHeaderTime, transform: cellWidth(30) },
    i18nHeaderDuration,
    { title: i18nHeaderStatus, transform: cellWidth('max') },
    i18nHeaderOutput
  ];
```

**Comment by kahboom** [comment]
**After PR feedback fix:**

Screenshot 2020-03-09 12 53 00

**Mobile:**

Screenshot 2020-03-09 12 53 38

**Pull request merged**